gio/tests/appmonitor.c

```diff
@@ -5,7 +5,6 @@ static gboolean
 create_app (gpointer data)
 {
   const gchar *path = data;
-  gchar *file;
```

Comment:
Thanks for catching this! It was indeed lost in a rebase. I pushed your commit to upstream master.